codice/ddf
# DDF-2507 Added MetacardType attribute registration to XmlInputTransformer

#### What does this PR do?
The attributes for metacards created using the XmlInputTransformer do not register the MetacardType Attributes. These attributes are now registered.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@brendan-hofmann 
@harrison-tarr 
@jrnorth 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@kcwire

#### How should this be tested?
Full build and ingest xml data with a numerical field. Check that the field can be searched on.

#### Checklist:
